tsdb/chunks/chunks.go

```diff
@@ -67,6 +67,20 @@ type Meta struct {
 	MinTime, MaxTime int64
 }
 
+// Iterator iterates over the chunk of a time series.
+type Iterator interface {
+	//// Seek advances the iterator forward to the given timestamp.
+	//// It advances to the chunk with min time at t or first chunk with min time after t.
+	//Seek(t int64) bool
```

Comment:
Is this meant to be commented out?

Reply:
Sorry! I just did not know if we really want Seek... I don't think we do. Removing.